src/main/resources/application.properties

```diff
@@ -37,3 +37,4 @@ consulting.types.json.schema.file=/schemas/consulting-type.json
 # CSRF token
 csrf.header.property=
 csrf.cookie.property=
+multitenancy.enabled=false
```

Comment:
I spoke also with Markus about this topic yesterday. Since the features in the project are turned off by default, we said it would make more sense to have it in turned off by default. i already added variables in kubernetes project which set it per default to true, so you will not notice any difficulties here.